nvtabular/dispatch.py

```diff
@@ -17,16 +17,20 @@
 import itertools
 from typing import Callable, Union
 
-import cudf
 import cupy as cp
 import dask.dataframe as dd
 import dask_cudf
 import numpy as np
 import pandas as pd
 import pyarrow as pa
 import pyarrow.parquet as pq
-from cudf.core.column import as_column, build_column
-from cudf.utils.dtypes import is_list_dtype
+
+try:
+    import cudf
+    from cudf.core.column import as_column, build_column
+    from cudf.utils.dtypes import is_list_dtype
+except ImportError:
+    cudf = None
```

Comment:
We'll also need to add support for cudf being None to all functions that use it in this file - and assume we're CPU only if not set.